docs/installation/setup.md

```diff
@@ -31,6 +31,43 @@ spec:
       globallyEnabled: true
 ```
 
+### Deployment of DNS controller manager
+
+If you are using Gardener version >= `1.54`, please make sure to deploy the DNS controller manager by 
```

Comment:
do you mean dns controller manager as part of the shoot dns service extension, here

Reply:
yes